### DISCUSSION TITLE
Support Package Registry Service

### DISCUSSION THREAD

**Discussion by tomisacat** [SwiftPackageIndex/SwiftPackageIndex-Server#3277] [Ideas]
As we all may know that the [drawback](https://github.com/swiftlang/swift-package-manager/issues/6062) of spm for a long-historical project, and fortunately Apple has provided an acceptable solution for this issue: [swift-evolution-0292-package-registry-service](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0292-package-registry-service.md). Do you have any plan for supporting this service in the future? Thanks!

**Comment by daveverwer** [comment]
This is certainly on our minds 👍 We have nothing to announce at the moment but having an open-source repository is going to be a huge step forward for the Swift community.

**Reply by tomisacat** [reply]
That's fantastic!
